pkg/ipam/metadata/manager.go

```diff
@@ -95,11 +96,34 @@ func splitK8sPodName(owner string) (namespace, name string, ok bool) {
 	return namespace, name, true
 }
 
-func (m *Manager) GetIPPoolForPod(owner string) (pool string, err error) {
+func deternimePoolByAnnotations(annotations map[string]string, family ipam.Family) (pool string, ok bool) {
```

Comment:
nit: determine is misspelled

Reply:
Yes, that's a typo. My mistake. Thank you.